README.md

```diff
@@ -40,7 +40,8 @@ The [OCI Image Media Types](media-types.md) document is a starting point to unde
 The high-level components of the spec include:
 
 * An archival format for container images, consisting of an [image manifest](manifest.md), an [image layout](image-layout.md), a set of [filesystem layers](layer.md), and [image configuration](config.md) (base OCI layer)
-* A process of hashing the image format for integrity and content-addressing (base OCI layer)
+* A [process of hashing the image format](descriptor.md) for integrity and content-addressing (base OCI layer)
+* A format for [storing CAS blobs and references to them](image-layout.md) (optional OCI layer)
```

Comment:
Somewhere around "Use of Hash as Content Addressable name for immutable containers" and "Archival Format".  But if we can't find it in the scope table, we should either update the scope table (my preference) or remove image-layout from the spec.